omod/src/main/java/org/openmrs/module/aijar/web/controller/AppointmentController.java

```diff
@@ -0,0 +1,28 @@
+package org.openmrs.module.aijar.web.controller;
+
+import java.util.Date;
+import java.util.List;
+
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@RequestMapping("/rest/ugandaemr/appointments")
+public class AppointmentController extends BaseRestController {
+
+    AdministrationService administrationService = Context.getAdministrationService();
+
+    @RequestMapping(method = RequestMethod.GET)
+    public int getPatients(@RequestParam(required = true) Date returnDate){
```

Comment:
@reagan-meant add a comma delimited list of encounterTypes which defaults to an empty string to allow the user to specify which encounter types are used in the count